incubator/etcd/templates/service.yaml

```diff
@@ -9,13 +9,13 @@ metadata:
     heritage: {{ .Release.Service | quote }}
     release: {{ .Release.Name | quote }}
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
-    component: "{{ .Release.Name }}-{{ .Values.Component }}"
+    component: "{{ .Release.Name }}-{{ .Values.component }}"
```

Comment:
The chart doesn't have multiple components. Why not remove the label? Instead you should add the `app` label.